static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -107,21 +102,21 @@ export default function FeedbackItem({feedbackItem}: Props) {
       <OverflowPanelItem>
         <Section title={t('Description')}>
           <Blockquote>
-            <pre>{feedbackItem.message}</pre>
+            <pre>{feedbackItem.metadata.message}</pre>
           </Blockquote>
         </Section>
 
         <Section icon={<IconLink size="xs" />} title={t('Url')}>
           <ErrorBoundary mini>
-            <TextCopyInput size="sm">{feedbackItem.url}</TextCopyInput>
+            <TextCopyInput size="sm">{'TODO'}</TextCopyInput>
           </ErrorBoundary>
         </Section>
 
         {feedbackItem.replay_id ? (
           <ReplaySection organization={organization} replayId={feedbackItem.replay_id} />
         ) : null}
 
-        <TagsSection tags={feedbackItem.tags} />
+        {/* <TagsSection tags={feedbackItem.tags} /> */}
```

Comment:
disabling until a followup PR when we also load in event details or whatever.
two steps forward, one step back here.